README.md

```diff
@@ -86,6 +86,7 @@ You can show your users you take security matters seriously and drive higher con
 * [sam-github](https://github.com/sam-github) - **Sam Roberts**
 * [SomeoneWeird](https://github.com/SomeoneWeird) - **Adam Brady**
 * [vdeturckheim](https://github.com/vdeturckheim) - **Vladimir de Turckheim**
+* [mcollina](https://github.com/mcollina) - **Matteo Collina**
```

Comment:
apologies on the nit pick, we order the users on alphabetical order of their github handles 😆